phpstan.neon.dist

```diff
@@ -1,10 +1,7 @@
+includes:
+        - vendor/phpstan/phpstan-symfony/extension.neon
```

Comment:
Though probably if we want to link this directly from the vendor - it may require the `phpstan/phpstan-symfony` pack, but probably only that one,  `phpstan/phpstan` is redundant I think.

Btw, I suppose this line fixed that ignored error message you deleted below, right?

Reply:
Yes it did :)